src/signed_data_ring.rs

```diff
@@ -0,0 +1,574 @@
+// Copyright 2015 Brian Smith.
```

Comment:
The `signed_data` prefix on this module name doesn't seem like it makes much sense at this point?